source/codegen/metadata/niscope/functions.py

```diff
@@ -2672,6 +2672,54 @@
         ],
         'returns': 'ViStatus'
     },
+    'GetStartTimestampInformation': {
+        'codegen_method': 'public',
```

Comment:
I was expecting this to be `codegen_method` `private` and then you don't need to include the `private` header. But that also excludes it from the `proto` API? Do we have some other method to exclude it from the compilation test but include it in the `proto`?

Reply:
I don't think so, not at the moment at least. I know we skip over them for the restricted services which is making me slightly lean towards that. I made a draft PR for what that would look like instead since it wasn't much more work than getting this one thrown together: #830 

I also tagged Harsha in a separate comment below to try and see which route to take.